src/resources/template.html

```diff
@@ -6,7 +6,7 @@
 		<meta name="viewport" content="width=device-width,initial-scale=1">
 		<meta name="mobile-web-app-capable" content="yes">
 		<meta name="apple-mobile-web-app-capable" content="yes">
-		<link rel="manifest" href="/manifest.json">
+		<link rel="manifest" href="<%= htmlWebpackPlugin.files.publicPath %>manifest.json">
```

Comment:
For me it works right now - 
`<link rel="manifest" href="<%= __webpack_public_path__ %>manifest.json">`,
why can't we use this?

Reply:
I wanted to stay away from `__webpack_public_path__` where possible. I think it is generally bad practice to be reading from that as it is mutable and not so reliavle. For `lib/entry.js` I used `__webpack_public_path__` because it is the only available reference in the context.

I understand the inconsistency, but let me know what you think!

Reply:
Yes, it is mutable, and this how webpack itself proposes to use this variable if the `publicPath`'s value is not known at compilation time, so you can even set it in the entry point.